source/Graphics.js

```diff
@@ -70,7 +70,7 @@ export default class Graphics
 						for(const q of loadQueue)
 						{
 							q.graphics.init();
-							q.cb();
+							cb();
```

Comment:
This would call the cb multiple times and would break queued components waiting for their callback. That's why loadQueue gets the cb pushed into the list as well as the graphics object.